automated/linux/igt/print-test-result.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+import argparse
+import sys
+import json
+
+
+def print_result(results):
+    try:
+        for test, content in results['tests'].iteritems():
+            print '************************************************************************************************************************************'
```

Comment:
Since you have all the results and logs in one place I would try to use LAVA_SIGNAL_STARTTC and LAVA_SIGNAL_ENDTC. This way LAVA will know about the test name, result and log. I think the proper sequence should be
<LAVA_SIGNAL_STARTTC test_case_name>
log output here. You can include all "print" statements you have below.
<LAVA_SIGNAL_TESTCASE TEST_CASE_ID=test_case_name RESULT=test_result>
<LAVA_SIGNAL_ENDTC test_case_name>

I didn't try so I'm not 100% sure.

Reply:
LAVA_SIGNAL_TESTCASE should come last. Here is the example:
https://validation.linaro.org/scheduler/job/1918367#L901

Reply:
I've revised the script.
Here is the test job
https://validation.linaro.org/scheduler/job/1918440#L2303